Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -271,6 +271,12 @@ class PlaceDetailsCardView: UIView {
         }
     }
 
+    func showEvent(atPlace place: Place) {
+        updateEventUI(forPlace: place)
+        self.setNeedsLayout()
+        self.layoutIfNeeded()
```

Comment:
They were here because I am going to add an animation to make the event appear nicer, and I don't really want the animation in the event view as it will need to be in the card view. I shall add comments to that effect here